scripts/pin_the_slow_way.py

```diff
@@ -36,6 +36,7 @@
           'icu': 'icu 56.*',
           'jpeg': 'jpeg 9*',
           'libblitz': 'libblitz 0.10|0.10.*',
+          'libevent': 'libevent 2.0.*',
```

Comment:
So we only have `libevent` version `2.0.22`. `defaults` only has `libevent` versions `2.0.20` and `2.0.21`. As such, I don't think that lower bound is necessary and this should be, for all practical concerns, fine.